Frontend/app/secure-test/secure-test.component.ts

```diff
@@ -0,0 +1,40 @@
+import { Component } from '@angular/core';
```

Comment:
This is a different naming convention from previous files with the '-' instead of camelCase